### PR TITLE
Docker enhancements

### PR DESCRIPTION
**Pull request by antonkap**
### Describe the Pull Request

```
Added possibility to use docker secrets inside container.
Added docker-compose_swarm.yml for deployment in docker swarm mode on single node.
Fixed postgres volume mount in docker-compose.yml
```

### Checklist for PR

- [ ] Run MobSF unit tests and lint `tox -e lint,test`
- [ ] Tested Working on Linux, Mac, Windows, and Docker
- [ ] Add unit test for any new Web API (Refer: `StaticAnalyzer/tests.py`)
- [ ] Make sure tests are passing on your PR [MobSF tests](https://github.com/MobSF/Mobile-Security-Framework-MobSF/actions)

### Additional Comments (if any)

```
DESCRIBE HERE
```

### CODE REVIEW

**Review by ajinabraham — COMMENTED** [review]
I am making some improvements with MobSF docker image that postgres will be supported out of the box without needing an additional docker image.

File: docker-compose_swarm.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    secrets:
+      - mobsfDB_password
+  mobsf:
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD_FILE=/run/secrets/mobsfDB_password
+      - POSTGRES_DB=mobsf
+      - POSTGRES_HOST=postgres
+      - MOBSF_API_KEY_FILE=/run/secrets/mobsf_api_key
+    image: mobsf:${MOBSF_IMAGE_VERSION:-v4.0.7}
```
[the remark this answers is not in the document]
Comment (antonkap) [reply docker-compose_swarm.yml:23]: Hi, @ajinabraham! It's just an image built locally(with postgres support, etc.)

**Comment by antonkap** [comment]
> I am making some improvements with MobSF docker image that postgres will be supported out of the box. Can you base your work off that image? #2439

Sure! I will update PR and change destination to Docker_postgres_2430 branch

**Comment by antonkap** [comment]
@ajinabraham since target branch was closed and original PR as well I've recreated it with changes you've mentioned: [#2440](https://github.com/MobSF/Mobile-Security-Framework-MobSF/pull/2440)

**Pull request closed without merging**